docs/index.md

```diff
@@ -44,6 +44,17 @@ Developers of this package are active on our [Zulip chat channel](https://images
   This is because it is transitional, and we have decided to put time into implementing other
   parts of the specification. We would welcome a pull request to add this functionality though!
 
+## Versioning
+
+`ome-zarr-models` has a 0.minor.micro versioning scheme where:
```

Comment:
You have `minor.micro` here but describe `major` and `minor` below?

Reply:
🤦 wooops, thanks for catching!